scripts/us_census/geojsons_low_res/generate_mcf.py

```diff
@@ -48,12 +64,14 @@ def generate_simple_geojsons(self, show=False):
             if int(geoid.split('/')[1]) > 100:
                 continue
             logging.info(f"Simplifying {geoid}...")
+            if eps is not None:
+                self.eps = eps
```

Comment:
This side effect should be documented.

It might be worth considering whether the function should just use `eps` if given and otherwise `self.eps` without any side effect.  If a person wants the side effect, the person should make an explicit call.